src/routes/MapExplorer/MapExplorer.js

```diff
@@ -48,6 +52,8 @@ export default class MapExplorer extends React.Component {
       showFeatureFilters: false,
       totalLoadedBoundingBox: null,
       viewport: {
+        width: 800,
```

Comment:
why is this needed?

Reply:
This allows the map to be zoomed in when loading a saved search from the user settings section. Otherwise, it isn't zoomed in at all.